openedx/features/course_experience/__init__.py

```diff
@@ -76,6 +76,8 @@
 # .. toggle_tickets: https://openedx.atlassian.net/browse/AA-36
 CALENDAR_SYNC_FLAG = CourseWaffleFlag(f'{WAFFLE_FLAG_NAMESPACE}.calendar_sync', __name__)  # lint-amnesty, pylint: disable=toggle-missing-annotation
 
+LEARNING_MFE_ENABLED = CourseWaffleFlag('learner_home_mfe.enabled', __name__)
```

Comment:
It looks like there is already a function called `course_home_mfe_progress_tab_is_active` which should probably be used here instead of adding a new waffle flag.

see lms/djangoapps/course_home_api/toggles.py

Reply:
Thanks for mentioning the existing function. Replaced it as you requested.

Reply:
Considering the lint error, do you suggest I add this import to the `ignore_imports` section of `[importlinter:contract:1]` in `setup.cfg` or is this a bad idea? 

By the way, is there any sort of doc on what steps must be taken before submitting a PR? I searched a little before this PR, studied [the test doc](https://docs.openedx.org/projects/edx-platform/en/latest/concepts/testing/testing.html), but `make lint-imports` wasn't mentioned there as far as I remember.

@feanil

Reply:
No, we shouldn't ignore it. `make lint-imports` is fairly new so it hasn't made it into the docs yet but it's a good idea to add it.  I wrote up https://github.com/openedx/edx-platform/issues/34390 to capture that idea.  I'll suggest a solution on the existing code for how to fix this.